kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -212,4 +309,33 @@
   .sign-in-error
     color: red
 
+  .suggestions
```

Comment:
Maybe instead of creating a dropdown, use the `ui-popover`? https://josephuspaye.github.io/Keen-UI/#/ui-popover